config/core/resources/channel.yaml

```diff
@@ -56,6 +56,9 @@ spec:
       properties:
         spec:
           properties:
+            serviceAccount:
+              type: string
+              description: "GCP service account used as credential to poll the Cloud Pub/Sub Subscription. The value of it must be a full format of gcp service account."
```

Comment:
nit: "GCP service account used to poll the Cloud Pub/Sub Subscription. The value of the service account must be a valid Google service account (see https://cloud.google.com/iam/docs/service-accounts). "

here and the others